core/domain/rte_component_registry_test.py

```diff
@@ -174,3 +175,25 @@ def test_default_rte_components_are_valid(self):
 
             self._validate_customization_arg_specs(
                 component._customization_arg_specs)  # pylint: disable=protected-access
+
+    def test_html_contains_all_imports(self):
+        """Test that the rich_text_components.html contains script-imports for
+        all directives of all RTE components."""
+
+        js_files_paths = []
```

Comment:
I suggest generating rtc_html_file_contents first, and then do all the assertIn checking within the for loop here. That way you don't need to maintain js_files_paths.

Reply:
Done.

Reply:
I kept this the same. Because I needed to sort the paths.